mesheryctl/pkg/utils/healthcheck.go

```diff
@@ -241,6 +241,20 @@ func AreAllPodsRunning() (bool, error) {
 	return true, nil
 }
 
+// CheckMesheryNsDelete waits for Meshery namespace to be deleted, returns (done, error)
+func CheckMesheryNsDelete() (bool, error) {
+	kubeClient, err := meshkitkube.New([]byte(""))
```

Comment:
```suggestion
	client, err := meshkitkube.New([]byte(""))
```
Minor edit for consistency across files. 

This is ready to merge.

Reply:
Done!